packages/features/eventtypes/components/tabs/limits/EventLimitsTab.tsx

```diff
@@ -640,6 +641,29 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
           />
         </div>
       </SettingsToggle>
+      <Controller
```

Comment:
I don't think Limits is the right tab, I feel like this needs to go in advanced

Reply:
since there are other settings related to slots and offsets, i thought Limits Tab was appropriate.